wolfssl/wolfcrypt/aes.h

```diff
@@ -223,6 +223,9 @@ struct Aes {
     byte   keyIdSet;
     byte   useSWCrypt; /* Use SW crypt instead of SE050, before SCP03 auth */
 #endif
+#ifdef WOLFSSL_MICROCHIP_TA100
+    uint16_t key_id;
```

Comment:
should be word16

Reply:
Why? used `uint16_t` to maintain consistency with what cryptoautholib uses.